lib/ssl/src/ssl_cipher.erl

```diff
@@ -933,8 +930,8 @@ signature_algorithm_to_scheme(#'SignatureAlgorithm'{algorithm = ?'id-RSASSA-PSS'
 %%   We return the original (possibly invalid) PadLength in any case.
 %%   An invalid PadLength will be caught by is_correct_padding/2
 %%
-generic_block_cipher_from_bin({3, N}, T, IV, HashSize)
-  when N == 0; N == 1 ->
+generic_block_cipher_from_bin(Version, T, IV, HashSize)
+  when Version == ?'SSL-3.0'; Version == ?'TLS-1.0' ->
```

Comment:
Here we should also be able to remove SSL-3.0

Reply:
We have a test that breaks if I remove it. 
So either the test is wrong or we should keep it...